### DISCUSSION TITLE
Dịch "duality" trong SVM

### DISCUSSION THREAD

**Discussion by quangnhat185** [mlbvn/handson-ml2-vn#85] [Terminology] [answered]
Hiện tại mình đang dịch `duality` là `đối ngẫu`. Anh @ngcthuong muốn bàn thêm về cách dịch từ này

Dưới đây là định nghĩa `duality` mình copy từ [Wiki](https://en.wikipedia.org/wiki/Duality_(optimization)):

`In mathematical optimization theory, duality or the duality principle is the principle that optimization problems may be viewed from either of two perspectives, the primal problem or the dual problem. The solution to the dual problem provides a lower bound to the solution of the primal (minimization) problem`

Mọi người cho ý kiến nhé @mlbvn/handson-ml2

**Comment by tiepvupsu** [comment, marked as answer]
Từ "đối ngẫu" được dùng nhiều trong các tài liệu khoa học kỹ thuật của VN rồi nhé, đặc biệt là trong Xử lý tín hiệu số với các biến đổi Fourier.

**Reply by ngcthuong** [reply]
Từ ngẫu ở đây có nghĩa là gì nhỉ?
Nếu được dùng nhiều trong tài liệu rồi thì mình cứ dùng thôi.
